components/Faq.vue

```diff
@@ -8,11 +8,11 @@
       :active-category="activeCategory"
       @clicked="scrollToCategory"
     />
-    <div class="FaqContent-Scroll-Area" @scroll.passive="debounce">
-      <div
-        class="FaqContent-Scroll-Length"
-        :style="{ height: `${lastFaq.top + lastFaq.height * 2}px` }"
-      >
+    <div
+      :class="{ 'FaqContent-Scroll-Area': getScreenWidth() > 400 }"
```

Comment:
Is there any reason of the 400 or 599 or 600 shows in the bottom code? I guess it is to distinguish mobile and desktop but I wonder why 3 types of width are shown in the same file.

Reply:
@kengoy the 599 should have been 600, but it doesn't really change much. Basically, at 600px, the sidenavigation disappears. Once it disappears, I'm telling the code to not add more space for scrolling. The sidenavigation relies on the extra space to allow the last FAQ to scroll all the way to the top.

As for the 400px, I'm removing the scroll area for mobile. On mobile, it becomes a long single page instead of having a scrollable section like on desktop view for the FAQ page.

Reply:
I changed the 599 to 600 to keep it consistent.